diff --git a/dist/rapier_wasm3d.js b/dist/rapier_wasm3d.js
index f506e35499d4d419a211ac6c82e1753198dca1b4..1a63573071b632114a7b490f53f2f383543b7ea8 100644
--- a/dist/rapier_wasm3d.js
+++ b/dist/rapier_wasm3d.js
@@ -5797,7 +5797,7 @@ module.exports.__wbindgen_memory = function() {
     return addHeapObject(ret);
 };
 
-const path = require('path').join(__dirname, 'rapier_wasm3d_bg.wasm');
+import path from './rapier_wasm3d_bg.wasm' with { type: "file" } 
 const bytes = require('fs').readFileSync(path);
 
 const wasmModule = new WebAssembly.Module(bytes);
